confgenerator/testdata/valid/linux/metrics-default_overrides_disable_all/golden_otel.conf.orig

```diff
@@ -0,0 +1,439 @@
+exporters:
```

Comment:
I think you have a merge error that created all these `.orig` files; they're not used by any of the tests AFAIK.

Reply:
Thank for catching this, removed those.